addon/initializers/setup-session.js

```diff
@@ -11,5 +11,7 @@ export default function setupSession(registry) {
     store = 'session-store:test';
     registry.register(store, Ephemeral);
   }
+
+  inject(registry, store, 'cookies', 'service:cookies');
```

Comment:
Ah, I see. But then we could just add sth. like 

```js
services: computed(function() {
  let owner = getOwner(this);

  return owner.lookup('service:cookies');
}),
```

to the cookie store so you don't have to set the service externally.